app/views/reviews/_form_edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for([@recipe, @review]) do |f| %>
+  <%= f.input :rating, collection: %w[1 2 3 4 5] %>
+  <%= f.input :comment, as: :text %>
+  <%= f.button :submit %>
+<% end %>
+<%= link_to "Back to the recipe", recipe_path %>
```

Comment:
@akramerin you should just have an auto re-direct tot the recipe instead of a secondary link under :)

Reply:
this is, if you ACCIDENTALLY came on the edit page and want to go back without editing. of course there is an auto direct if you choose to edit it. but i can fix it, it is kinda redundant.

Reply:
I would call it return then ;)